app/navigation/Navigation.scala

```diff
@@ -48,7 +48,9 @@ class Navigation @Inject()(dateTimeHelper: DateTimeHelper,
     WhyWasTheReturnSubmittedLateAgentPage -> ((_, request, _) => routeToMakingALateAppealOrCYAPage(request, CheckMode)),
     ReasonableExcuseSelectionPage -> ((_, request, _) => routeToMakingALateAppealOrCYAPage(request, CheckMode)),
     WhenDidThePersonDiePage -> ((_, request, _) => routeToMakingALateAppealOrCYAPage(request, CheckMode)),
-    OtherRelevantInformationPage -> ((_, request, _) => routeToMakingALateAppealOrCYAPage(request, CheckMode))
+    OtherRelevantInformationPage -> ((_, request, _) => routeToMakingALateAppealOrCYAPage(request, CheckMode)),
+    UploadFirstDocumentPage -> ((_, _, _) => routes.OtherReasonController.onPageLoadUploadList(CheckMode)),
+    UploadAnotherDocumentPage -> ((answer, _, _) => routeToAdditionalUploadOrCYAPage(CheckMode, answer))
```

Comment:
I think once the user has uploaded another document and they are in CheckMode, that they should be routed to the upload list page?

Reply:
A case for FileListPage should be added so it then calls  routeForUploadList